pqkmeans/clustering/pqkmeans.py

```diff
@@ -1,37 +1,52 @@
 import typing
 import numpy
 import sklearn
-
+import collections
 import _pqkmeans
 
+PQKMeansSavedata = collections.namedtuple('PQKMeansSavedata',
+                                          ('encoder', 'k', 'iteration', 'verbose', 'cluster_centers'))
+
 
 class PQKMeans(sklearn.base.BaseEstimator, sklearn.base.ClusterMixin):
-    def __init__(self, encoder, k, iteration=10, verbose=False, init_centers=None):
+    def __init__(self, encoder, k, iteration=10, verbose=False):
         super(PQKMeans, self).__init__()
-        if init_centers is None:
-            init_centers = []
-        self._impl = _pqkmeans.PQKMeans(encoder.codewords, k, iteration, verbose, init_centers)
+        self.encoder = encoder
+        self._impl = _pqkmeans.PQKMeans(self.encoder.codewords, k, iteration, verbose)
```

Comment:
Sorry, I removed `init_centers` to unify the way of loading trained instance. @insikk